src/main/java/org/usfirst/frc4048/common/loggingv2/CommandLogger.java

```diff
@@ -0,0 +1,53 @@
+package org.usfirst.frc4048.common.loggingv2;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.*;
+
+public class CommandLogger {
+    private final LinkedHashMap<Command, Queue<Boolean>> toLogCommandStatus = new LinkedHashMap<>();
```

Comment:
(Not sure, can't check right now) but `Command` probably does not implement `equals` and `hashCode` therefore is not a good candidate to be put in a map (though we may get away with identity equality initially)

Reply:
At any rate, a List<Command, Boolean> may be better suited for this

Reply:
Regarding your first comment, I can definitely implement those methods (Command does not).
Regarding your second command, because of the multiple state values per tick problem I talked about in another comment, it would have to be a `List<Command, Queue<Boolean>>`. However we would not get the "instant" lookup times of Hashmap when iterating to find the entry for each command.

Reply:
Discussed.